src/lib/pages/tx-details/components/tx-message/msg-receipts/index.tsx

```diff
@@ -1607,8 +1607,8 @@ export const generateReceipts = (
           value: details.token_in_denom,
         },
         {
-          title: "Share Out Amount",
-          value: details.share_out_amount,
+          title: "Share Amount Out",
```

Comment:
The field name of `MsgJoinSwapShareAmountOut` share_out_amount so I think it should stick with that name